fedoracommunity/server/packages.py

```diff
@@ -0,0 +1,3 @@
+def count_packages(packages):
+    subpackages = sum([len(package.subpackages) for package in packages])
+    return len(packages) + subpackages
```

Comment:
But if the search "foo" founds a package "foo", which has subpackage python-foo and subpackage bar then it will print 3 instead of 2. Which is wrong. Right?

Reply:
Is it though? There will be printed 3 rows in the table, so I would say it is appropriate to print that "3 results were found".

Reply:
Hmmm, true.

https://apps.fedoraproject.org/packages/s/python-copr

This prints two rows in the table and says "1 results". Which is weird to me but I will implement it the same way. Thank you for pointing that out.